source/slang/slang-emit-c-like.cpp

```diff
@@ -1849,6 +1882,27 @@ void CLikeSourceEmitter::emitIntrinsicCallExprImpl(
     }
 }
 
+void CLikeSourceEmitter::_emitCallArgList(IRCall* inst)
+{
+    m_writer->emit("(");
+    UInt argCount = inst->getOperandCount();
+    for (UInt aa = 1; aa < argCount; ++aa)
+    {
+        auto operand = inst->getOperand(aa);
+        if (as<IRVoidType>(operand->getDataType()))
+            continue;
```

Comment:
Ideally we should have an earlier IR pass that is responsible for eliminating all `void`-type parameters, local variables, fields, etc. rather than the current logic we have in the emit passes to tiptoe around `void`.